<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefinementDialog</class>
 <widget class="QDialog" name="RefinementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Refinement</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Working directory:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="workingDirLineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="workingDirPushButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Model (PDB):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="modelLineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="modelPushButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Data:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="dataLineEdit"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="dataPushButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Refinement type:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="refmac5RadioButton">
        <property name="text">
         <string>Refmac5</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rigidBodyRadioButton">
        <property name="text">
         <string>Rigid Body</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="shelxRadioButton">
        <property name="text">
         <string>SHELX</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>B-Factor treatment</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="isotropicRadioButton">
        <property name="text">
         <string>Isotropic</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="anisotropicRadioButton">
        <property name="text">
         <string>Anisotropic</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Weight:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="weightSpinBox">
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Number of cycles:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="numCyclesSpinBox">
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="waterPickingCyclesSpinBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="waterPickingCheckBox">
       <property name="text">
        <string>Water picking cycles:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Optional items</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="maxResolutionCheckBox">
        <property name="text">
         <string>Max resolution</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDoubleSpinBox" name="maxResolutionSpinBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="tlsSpecificationCheckBox">
        <property name="text">
         <string>TLS Specification (tls)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="tlsSpecificationLineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="tlsSpecificationPushButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="dictionaryCheckBox">
        <property name="text">
         <string>Dictionary</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="dictionaryLineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="dictionaryPushButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>workingDirLineEdit</tabstop>
  <tabstop>workingDirPushButton</tabstop>
  <tabstop>modelLineEdit</tabstop>
  <tabstop>modelPushButton</tabstop>
  <tabstop>dataLineEdit</tabstop>
  <tabstop>dataPushButton</tabstop>
  <tabstop>rigidBodyRadioButton</tabstop>
  <tabstop>refmac5RadioButton</tabstop>
  <tabstop>shelxRadioButton</tabstop>
  <tabstop>isotropicRadioButton</tabstop>
  <tabstop>anisotropicRadioButton</tabstop>
  <tabstop>weightSpinBox</tabstop>
  <tabstop>numCyclesSpinBox</tabstop>
  <tabstop>waterPickingCheckBox</tabstop>
  <tabstop>waterPickingCyclesSpinBox</tabstop>
  <tabstop>maxResolutionCheckBox</tabstop>
  <tabstop>maxResolutionSpinBox</tabstop>
  <tabstop>tlsSpecificationCheckBox</tabstop>
  <tabstop>tlsSpecificationLineEdit</tabstop>
  <tabstop>tlsSpecificationPushButton</tabstop>
  <tabstop>dictionaryCheckBox</tabstop>
  <tabstop>dictionaryLineEdit</tabstop>
  <tabstop>dictionaryPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RefinementDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RefinementDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
